mex/common/identity/__init__.py

```diff
@@ -1,12 +1,10 @@
 from mex.common.identity.base import BaseProvider
 from mex.common.identity.models import Identity
 from mex.common.identity.registry import get_provider, register_provider
-from mex.common.identity.types import IdentityProvider
```

Comment:
Why is this completely removed instead of replaced by `from mex.common.types import IdentityProvider`?